src/library_glfw.js

```diff
@@ -1263,6 +1285,8 @@ var LibraryGLFW = {
   glfwGetWindowMonitor: function(winid) {
     var win = GLFW.WindowFromId(winid);
     if (!win) return 0;
+    if (!GLFW.active.fullscreen) return 0;
```

Comment:
This might be safer to also read `if (!GLFW.active || !GLFW.active.fullscreen) return 0;` like in the drag and drop API?